src/pretalx/cfp/views/event.py

```diff
@@ -34,6 +36,19 @@ def get_context_data(self, **kwargs):
         context['has_sneak_peek'] = self.request.event.submissions.filter(
             is_featured=True
         ).exists()
+        # Extract track= and submission_type= from query string.
+        # Only used first value if key is used multiple times.
+        track = self.request.GET.get('track', None)
+        submission_type = self.request.GET.get('submission_type', None)
+        params = []
+        if track:
+            params.append(f'track={track}')
+        if submission_type:
+            params.append(f'submission_type={submission_type}')
+        if len(params) > 0:
+            context['submit_qs'] = f'?{"&".join(params)}'
```

Comment:
Letting `params` be a dict and using `urllib.urlencode(params)` would make sure all content is properly escaped. (Please test this with a track with special characters, such as umlauts.